dev/io.openliberty.microprofile.reactive.streams.operators30.internal/test/src/io/openliberty/microprofile/reactive/streams/test/UtSubscriber.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.reactive.streams.test;
+
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+/**
+ *
+ */
+final class UtSubscriber implements Subscriber<String> {
```

Comment:
I can't see any references to this class.

Reply:
er, ok!

Reply:
Sorry, this was at the end of the review and my comments got a bit terse.

This is a utility class but I don't think any of the tests are actually using it. If so we could delete it.